ajax/libs/response.js/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "response.js",
+  "version": "0.9.1",
+  "filename": "response.min.js",
+  "description": "Response is an experimental jQuery/Ender/Zepto plugin that gives web designers tools for building responsive websites. It can dynamically swap content based on breakpoints and data attributes.",
+  "homepage": "http://responsejs.com/",
```

Comment:
This domain name seems to be expired, we should remove it.

Reply:
@maruilian11 I can access this website.
Can't you?
Thank you.